scripts/migrate_v237_limits.py

```
@@ -0,0 +1,77 @@
+dry_run = args.dry_run
+
+print("---------------------------------------------------------")
+print("V2.37 limits migration script")
+if dry_run:
+    print("dry-run=" + str(dry_run) + " -> will run without modifying anything (test mode)")
+if constant.DEV:
+    print("DEV=" + str(constant.DEV) + " -> hostnames configured for a local execution (172.17.0.1:xxxx)")
+print("\n")
+
+# Check network-store-server
+if not check_server_status(constant.NETWORK_STORE_SERVER_HOSTNAME): sys.exit()
```

Ruff (0.15.13)

[error] Multiple statements on one line (colon)

(E701)